tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html

```diff
@@ -104,14 +104,14 @@
         stroke: #0f0    !important;
       }
 
-      #sessionGroupValues {
-        background: black;
-        border-radius: 5px;
-        border: 2px solid black;
-        padding: 8px;
-        box-shadow: 2px 2px #777;
-        max-width: 100%;
-        max-height: 100%;
+      #container {
+        width: 100%;
+        height: 100%;
```

Comment:
nit: sort a-z

Reply:
Done. Though, I don't think I'm doing this in other places. Is there a style-guideline for sorting members of a CSS group somewhere?

Reply:
Ah sorry, I should have linked a rule here. This is the rule: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

or go/css-style internally.